### PR TITLE
Adding duplicate files in a pulp repo(TestCase 1).

### PR DESCRIPTION
**Pull request by ragabala**
This commit takes care of testing the scenario in which different files
with the same file name gets synced to the same repo and how it
is handled.

refer #4738

### CODE REVIEW

File: pulp_file/tests/functional/api/test_sync.py [hunk 1]
```diff
@@ -148,3 +149,50 @@ def do_test(self, url):
         with self.assertRaises(TaskReportError) as context:
             sync(self.cfg, remote, repo)
         return context
+
+
+class SyncUploadDuplicateFileRepoTestCase(unittest.TestCase):
+    """Sync multiple remptes containing duplicate files."""
```
Comment (kersommoura) [inline pulp_file/tests/functional/api/test_sync.py]:
```suggestion
    """Sync multiple remotes containing duplicate files."""
```

File: pulp_file/tests/functional/api/test_sync.py [hunk 2]
```diff
@@ -148,3 +149,50 @@ def do_test(self, url):
...
+        """Create class-wide variables."""
+        cls.cfg = config.get_config()
+        cls.client = api.Client(cls.cfg, api.json_handler)
+
+    def test_duplicate_file_sync(self):
+        """Sync a repository with remotes containing same file names.
+
+        This test does the following.
+
+        1. Create a repoository in pulp.
```
Comment (kersommoura) [inline pulp_file/tests/functional/api/test_sync.py]:
```suggestion
        1. Create a repository in pulp.
```

File: pulp_file/tests/functional/api/test_sync.py [hunk 3]
```diff
@@ -148,3 +149,50 @@ def do_test(self, url):
         with self.assertRaises(TaskReportError) as context:
             sync(self.cfg, remote, repo)
         return context
+
+
+class SyncUploadDuplicateFileRepoTestCase(unittest.TestCase):
```
Comment (kersommoura) [inline pulp_file/tests/functional/api/test_sync.py]:
```suggestion
class SyncDuplicateFilesRepoTestCase(unittest.TestCase):
```

**Comment by ragabala** [comment]
Thanks @kersommoura ,

Have made those changes. Let me know if this looks good.

**Pull request merged**